src/Appwrite/Migration/Version/V21.php

```diff
@@ -36,6 +36,11 @@ public function execute(): void
 
         Console::info('Migrating Documents');
         $this->forEachDocument([$this, 'fixDocument']);
+
+        if ($this->project->getInternalId() !== 'console') {
+            Console::info('Migrating Buckets');
+            $this->migrateBuckets();
+        }
```

Comment:
Might be good to do the bucket first before iterating over all documents, doing the container level stuff first.